index.html

```diff
@@ -12340,7 +12454,7 @@ <h3>States and Properties</h3>
 				</ul>
 				</li>
 				<li>Otherwise, ignore the value and treat the property as not present.</li>
```

Comment:
Change "property"
to "attribute".
(for consistency with other points in this list).

Reply:
happy to accept a PR cleaning these up but I don't want to make this any larger than it has to be.

Reply:
Agreed. I looked into writing a cleanup PR, but it's a bit of a pandora's box, so leaving it for now.
Resolving this conversation.